Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -377,6 +377,8 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
                                         View adView = vungleMrecAd.renderNativeView();
                                         if (adView != null) {
                                             isLoadSuccess = true;
+                                            // Honoring the server dimensions forces the WebView to be the size of the MREC
+                                            AdViewController.setShouldHonorServerDimensions(layout);
```

Comment:
This line forces our MREC ad view to the request size like 300x250(by default). But the request size maybe not a standard size for us like 345x267 (random size) , as a result, our mrec ad display with 345x267 not 300x250.
1. I tested with layout params similar to AdMob adapter's implementation but can't work for MoPub.
2. I created VungleMrec file similar to VungleBanner in native SDK and it works well without this line.
So, we'd better fix it on sdk side.

Reply:
Similar to VungleBanner, add mrecViewWrapper RelativeLayout to force return Vungle standard MREC ad.